### ISSUE TITLE
Audio Page Enhancements 

### ISSUE THREAD

**Issue by AlyafeiAli** [SpecialProjectADU/Site#88]
### These comments are from Dr. Mourad.  

- Header replacement
- Full redesign
- Add audio title and download button
- Add filters (Like BrowseVideos)

Responsibility: @Alyazi010 and her team.  
Write your questions in the comments and mention Mouatism.

![image](https://github.com/SpecialProjectADU/Site/assets/52626961/11fecf89-ed4f-4164-8c33-fa7da81471e9)

**Comment by Alyazi010** [comment]
what do you mean by redesign? @Moutasim02

**Comment by Moutasim02** [comment]
As per Dr. Murad's decision, the only search bar should be in the top(header), also use the same layout as other pages like Browse Video and Photos, so the design should be similar to the other designs.

**Comment by Moutasim02** [comment]
@Alyazi010 Its basically just the same as BrowsePhotos and BrowseVideos

**Comment by AlyafeiAli** [comment]
> what do you mean by redesign? the audio has two sections, first is the page where it contains the search bar, and the second is the containers where you play the audio @Moutasim02

From what I understood from Moutasim, you should get rid of the first section you mentioned here (the one with the searchbar) and replace it with the universal navbar and the search filters (both can be found in BrowsePhotos & BrowseUsers).  
As for the second section, you should change its layout and make it look similar to BrowseVideos (sort of), idk how you're supposed to achieve that but that's what the Dr is asking you to do. + Each audio file should have a title and a download button.  ( Replace the current design with bootstrap cards grid maybe or something like that ).

Please correct me if I misunderstood something, @Moutasim02

**Comment by Moutasim02** [comment]
Yes exactly like that, don't forget a brief description for each card too

**Comment by Alyazi010** [comment]
guys everything is clear to me whats need to be changed but the problem is that i need the code from where are you referring to? which page there's no such BrowsePhotos or Browsevideos  @Moutasim02
